src/luaconnector/common.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <thread>
+#include <mutex>
+#include <vector>
+#include <string>
+#include <cstdio>
+#include <utility>
+#include <deque>
+#include <chrono>
+
+#ifdef _WIN32
+#ifndef _WIN32_WINNT
+#define _WIN32_WINNT 0x0A00
```

Comment:
I explicitly skipped out on defining this in the Makefile so each part can do whatever (yes, we get warnings for that). If we really want to set it, set it in the Makefile, however i don't think you use any feature >WinXP, so the value for Win10 is definitely wrong. We currently target "Everything that works with mingw-w64". I know that Msys only supports 8.1 and newer, but plain mingw might work even further back.